### PR TITLE
Update activity test according to gh-4407

### PR DESCRIPTION
**Pull request by mkralik3**
//test: `@activity and not @gh-4192`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/IntegrationFlowView.java [hunk 1]
```diff
@@ -98,9 +98,11 @@ public void clickRandomTrash() {
         this.getElementRandom(Element.DELETE).shouldBe(visible).click();
     }
 
-    public void clickAddStepLink(int pos) {
+    public void clickAddStepLink(int pos) throws InterruptedException {
+        Thread.sleep(2000);
         if (isExpanded()) {
             $(Button.Collapse).click();
+            Thread.sleep(2000);
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/IntegrationFlowView.java]: use `TestUtils.sleepIgnoreInterrupt` and you don't have to mess with the exception

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**